corehq/preindex/tasks.py

```diff
@@ -2,9 +2,16 @@
 from celery.task.base import periodic_task
 
 from corehq.preindex.accessors import index_design_doc, get_preindex_designs
+from corehq.util.decorators import serial_task
+from django.conf import settings
 
 
-@periodic_task(run_every=crontab(minute=0), queue='background_queue')
+@periodic_task(run_every=crontab(minute='*/5'), queue=settings.CELERY_PERIODIC_QUEUE)
+def run_continuous_indexing_task():
+    preindex_couch_views.delay()
+
+
+@serial_task('couch-continuous-indexing', timeout=60 * 60, queue=settings.CELERY_PERIODIC_QUEUE, max_retries=0)
```

Comment:
guessing it's to try and prevent a reindex from starting before the previous one has finished. i think that was still a possibility in the previous version, and now more of a possibility since the interval is shorter